luda-editor/luda-editor-client/src/app/types/meta.rs

```diff
@@ -10,6 +11,7 @@ use wasm_bindgen_futures::spawn_local;
 pub struct Meta {
     pub subtitle_language_minimum_play_duration_map: HashMap<Language, Time>,
     pub subtitle_language_play_duration_per_character_map: HashMap<Language, Time>,
+    pub subtitle_specific_text_token_play_duration_map: LinkedHashMap<String, Time>,
```

Comment:
I used `LinkedHashMap` instead `HashMap` for ordering
`LinkedHashMap` keeps order of entry